lib/spack/spack/cmd/install.py

```diff
@@ -43,11 +45,13 @@
 description = "Build and install packages"
 
 
+
+
 def setup_parser(subparser):
     subparser.add_argument(
         '--only',
         default='package,dependencies',
-        dest='things_to_install',
+        dest='only_str',
```

Comment:
It might be me, but I don't get what the name `only_str` means in this context. Why did you change the name of this variable?